ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/intended/structured_configs/port-channel-short-esi.yml

```diff
@@ -0,0 +1,33 @@
+hostname: port-channel-short-esi
+service_routing_protocols_model: multi-agent
+vlan_internal_order:
+  allocation: ascending
+  range:
+    beginning: 1006
+    ending: 1199
+vrfs:
+- name: MGMT
+  ip_routing: false
+management_api_http:
+  enable_vrfs:
+  - name: MGMT
+  enable_https: true
+ip_igmp_snooping:
+  globally_enabled: true
+ethernet_interfaces:
+- name: Ethernet10
+  peer: server01_ESI
+  peer_interface: Eth1
+  peer_type: server
+  port_profile: TENANT_A_B
+  description: server01_ESI_Eth1
+  shutdown: false
+  type: port-channel-member
+  channel_group:
+    id: 10
+    mode: active
+port_channel_interfaces:
+- name: Port-Channel10
+  description: server01_ESI_PortChanne1
+  type: switched
+  shutdown: false
```

Comment:
Where is the esi ? There should be an ethernet_segment popping up here. You should fix the python code. The deprecated vars MUST still generate the same config as before with only a warning to the user on the upcoming removal (which you have already done in schema)